libs/base/utilities/src/decorators/hydratable/hydratable.ts

```diff
@@ -11,62 +11,71 @@ const DEFER_HYDRATION = Symbol('deferHydration');
 const HYDRATION_CALLS = Symbol('hydrationCalls');
 export const HYDRATE_ON_DEMAND = '$__HYDRATE_ON_DEMAND';
 export const HYDRATING = '$__HYDRATING';
+export const hydratableAttribute = 'hydratable';
 
 export interface PatchableLitElement extends LitElement {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/no-misused-new
   new (...args: any[]): PatchableLitElement;
   _$needsHydration?: boolean;
+  hasSsr: boolean;
```

Comment:
I'd not pollute componet class with this arbitrary variable. We could use symbol, but actually I'm wondering if we really need it, and can get away withuot having it on the class?